backend/app/libs/coingecko/coins.py

```diff
@@ -71,6 +73,8 @@ async def get_coin_hist_price(
         sleep(5)
         try:
             prices = resp.json().get("prices")
+            if prices is None:
```

Comment:
This doesn't allow to store in `db` when prices is `None`. That means we're not caching http calls for these tokens.

Reply:
> For a given token which Coingecko response is always misbehaving, we should definitely cache it.

I see what you mean now. This error seems to be popping up for spam tokens that passed our `quote < 1e18` filter (in testing, [ABSHIBA.com](https://etherscan.io/token/0x265befe2b1a0f4f646dea96ba09c1656b74bda91) was giving this issue).

Fair to say that isn't a temporary case, but I agree that we should cache it. I think a good compromise is to always cache and put an expiry (1 day maybe), return `None` when `prices is None`